dev/com.ibm.websphere.javaee.annotation.1.3/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>com.ibm.websphere.javaee.annotation.1.2</name>
```

Comment:
this should be 1.3

Reply:
Have amended the pull request to change the version as requested.